cmd/kyverno/main.go

```diff
@@ -87,7 +90,8 @@ func main() {
 	}
 
 	klog.InitFlags(nil)
-	log.SetLogger(klogr.New())
+	log.SetLogger(klogr.NewWithOptions(klogr.WithFormat(klogr.FormatKlog)))
```

Comment:
I think you are targeting text mode right ? In this case you are not calling the correct constructor.

Reply:
Yeah, the option is to have a better JSON output. Without it klogr would serialize the input by itself which affect the output of a JSON log by adding forward slashes to it.
cc: https://kubernetes.slack.com/archives/C020CCMUEAX/p1663089940802399?thread_ts=1663077396.632879&cid=C020CCMUEAX

Reply:
Please fix this.